keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -56,6 +64,28 @@ Consider the following scenario:
 
 In this case, the administrator would like the evicted workload to be requeued as soon as possible on the newly available capacity.
 
+#### Story 2
+
+In the story 1 scenario, when we set `waitForPodsReady.requeuingStrategy.timestamp=Creation`, 
+the workload endlessly or repeatedly can be put in front of the queue after eviction in the following eviction reasons:
+
+1. The workload don't have the proper configurations like image pull credential and pvc name, etc.
+2. The cluster can meet flavorQuotas, but each node doesn't have the resources that each podSet requests.  
+
+Specifically, the second reason will often occur in the workloads required gpus.
+Given that the workload with a request of 2 gpus is submitted to the cluster that
+has 2 worker nodes with 4 gpus, and 3 gpus are used (which means 1 gpu is free in each node),
+the workload will be repeatedly evicted because of the lack of resources in each node even though the cluster has enough capacities.
+
+In this case, to avoid rapid repetition of the admission and eviction cycle,
+the administrator would like to use an exponential backoff mechanism and add a maximum number of retries.
+
+#### Story 3
+
+In the story 3 scenario, after the evicted workload reached the maximum retry condition, 
+we want to easily requeue the workload to the queue without recreating the job. 
```

Comment:
And what is the high level solution for this?

Reply:
Maybe put in the story something like:

"This is possible if the Workload is deactivated (active=false), as opposed to deleting it".

Reply:
SGTM

Reply:
Done.